### PR TITLE
Rejig the docs

### PR DESCRIPTION
**Pull request by kitallis**
**Story card:** [ch2176](https://app.clubhouse.io/simpledotorg/story/2176/server-housekeeping)

Remove deprecated docs and clean-up the organization of `docs/`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -318,8 +321,7 @@ bundle exec cap india:staging deploy:rake task=db:seed
 
 ### Deployment Resources
 
-* [Deployment instructions for Simple Server](doc/deployment.md)
```
Comment (kitallis) [inline README.md:321]: I've kept this file in `wiki` but removed it from this link. I'm not certain if that `deployment.md` doc is still valuable.

Comment (harimohanraj89) [reply README.md:321]: A deployment doc is in general valuable, but this one is deprecated. More up-to-date docs are in the readme: https://github.com/simpledotorg/simple-server#deployment

Wanna link to that instead?

**Review by harimohanraj89 — APPROVED** [review]
One must do away with the old before bringing in the new. Changes seem minor enough - lgtm.

File: README.md (hunk 1 above)
Comment (kitallis) [reply README.md:321]: Ah, in which case I'll just delete this file for now. The README one is good enough as is.

**Pull request merged**